pom.xml

```diff
@@ -1,23 +1,25 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>edu.tamu</groupId>
-  <artifactId>catalog-service</artifactId>
+  <groupId>edu.tamu.catalog</groupId>
+  <artifactId>catalog-parent</artifactId>
   <version>2.2.0</version>
-  <name>Catalog-Service</name>
-  <description>A service for interfacing with Library Catalogs</description>
+
+  <name>Catalog Parent</name>
+  <description>A service and a domain for interfacing with Library Catalogs</description>
 
   <parent>
```

Comment:
I would like to bring up to the team whether we should use weaver parent. CatalogService will not use anything from it and we should be getting away from the bad practices it prescribes.

Reply:
If we can drop it without consequence here, I would say we do so.